build/mdn-browser-compat-data-importer.js

```diff
@@ -0,0 +1,115 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+const bcd = require('mdn-browser-compat-data')
+
+function addBrowserCompatDataToProperties(atdirectives, pseudoclasses, pseudoelements, properties) {
+  atdirectives.forEach(item => {
+    if (bcd.css['at-rules'][item.name.slice(1)]) {
+      const matchingBCPItem = bcd.css['at-rules'][item.name.slice(1)] 
```

Comment:
there's also a bunch of `BCP` in this file :)

Reply:
@connorshea You are right. Originally wanted to call it browser-compat-property.